LondonDataServices.IDecide.Core.Tests.Unit/Services/Foundations/Notifications/NotificationServiceTests.SendCodeNotification.Validations.cs

```diff
@@ -77,14 +77,9 @@ public async Task ShouldThrowValidationExceptionOnSendCodeNotificationIfNotifica
                 },
                 Decision = new Decision
                 {
-                    DecisionChoice = invalidText,
                     ResponsiblePersonGivenName = invalidText,
```

Comment:
Why has this been removed.  They could be potential values on the replacement dictionary?